lib/node_modules/@stdlib/array/complex64/docs/types/index.d.ts

```diff
@@ -728,6 +728,26 @@ declare class Complex64Array implements Complex64ArrayInterface {
 	*/
 	indexOf( searchElement: ComplexLike, fromIndex?: number ): number;
 
+	/**
+	* Returns string representations of array elements.
+	*
+	* @param separator - value separator (default: ',')
+	* @returns string
+	*
+	* @example
+	* var arr = new Complex64Array( 2 );
+	*
+	* arr.set( [ 1.0, 1.0 ], 0 );
+	* arr.set( [ 2.0, 2.0 ], 1 );
+	*
+	* var str = arr.join( '/' );
+	* // returns '1 + 1i/2 + 2i'
+	*
+	* str = arr.join();
+	* // returns '1 + 1i,2 + 2i'
```

Comment:
```suggestion
	* var str = arr.join();
	* // returns '1 + 1i,2 + 2i'
	*
	* str = arr.join( '/' );
	* // returns '1 + 1i/2 + 2i'
```

Start with the base/default case and then move to specialized case.